earthmover/earthmover.py

```diff
@@ -130,6 +137,14 @@ def load_project_configs(self, filepath: str):
 
         return configs
 
+    def inject_cli_overrides(self, configs, prefix=None):
+        # parse self.overrides into configs:
+        for i in range(0, len(self.overrides), 2):
+            key = self.overrides[i]
+            if not prefix or key.startswith(prefix):
+                value = self.overrides[i+1]
+                configs.set_path(key.lstrip(prefix), value)
+        return configs
```

Comment:
This walks through each (key, value) pair in `self.overrides` and updates the value (with `set_path()`).